db/models.js

```diff
@@ -45,6 +46,30 @@ exports.fetchReview = (reviewId) => {
 
 }
 
-exports.retrieveEndpoints = () => {
-    
-}
\ No newline at end of file
+exports.fetchCommentsById = (queryId) => {
+    const sqlInsertion = [queryId]
+    const sqlQuery = `SELECT * FROM comments
+    WHERE review_id = $1;`
+    return doesCategoryExist('reviews', 'review_id', queryId)
```

Comment:
I think using a util for this may be a bit unnecessary, makes it less readable.

Reply:
does it not make it re-usable in case we have to use it in future tasks?

Reply:
I understand what Joe is saying here but I think a util function is fine. I would reconsider your current util function though. Though it's intended to be reused with those three parameters allowing for different tables, columns and ids, your function can only currently query the reviews table due to the statement used inside. For what it's worth, I don't think we reuse this anywhere in the next tasks, but having a reusable function is a really great idea :)